src/main/java/com/nike/cerberus/operation/composite/DeleteCerberusEnvironmentOperation.java

```diff
@@ -0,0 +1,76 @@
+package com.nike.cerberus.operation.composite;
+
+import com.google.common.collect.ImmutableList;
+import com.nike.cerberus.command.composite.DeleteCerberusEnvironmentCommand;
+import com.nike.cerberus.command.core.DeleteStackCommand;
+import com.nike.cerberus.domain.EnvironmentMetadata;
+import com.nike.cerberus.service.ConsoleService;
+
+import javax.inject.Inject;
+import java.util.LinkedList;
+import java.util.List;
+
+import static com.nike.cerberus.domain.environment.Stack.*;
+
+public class DeleteCerberusEnvironmentOperation extends CompositeOperation<DeleteCerberusEnvironmentCommand> {

```

Comment:
nit: maybe this should just be DeleteEnvironmentOperation/Command

Reply:
sure we can rename this latter if you want.